main.py

```diff
@@ -166,3 +198,22 @@ def handle_stats_received(self, event):
     def handle_stats_reply_received(self, replies_flows):
         """Update the set of flows stats"""
         self.flows_stats_dict.update({flow.id: flow for flow in replies_flows})
+
+    @listen_to('kytos/of_core.table_stats.received')
+    def on_table_stats_received(self, event):
+        """Capture table stats messages for OpenFlow 1.3."""
+        self.handle_table_stats_received(event)
+
+    def handle_table_stats_received(self, event):
+        """Handle table stats messages for OpenFlow 1.3."""
+        if 'replies_tables' in event.content:
```

Comment:
`kytos/of_core.table_stats.received` is expected to always have a `replies_table` if it doesn't then the event is badly formatted and you can let it crash, so no need for this conditional here.